server/config.go

```diff
@@ -516,6 +519,8 @@ func (c *config) Clone() (Config, error) {
 		Leaderboard:      &configLeaderboard,
 		Matchmaker:       &configMatchmaker,
 		IAP:              &configIAP,
+		Satori:           &configSatori,
+		GoogleAuth:       &configGoogleAuth,
```

Comment:
No need to clone this value because it will be redacted here https://github.com/heroiclabs/nakama/blob/master/server/console_config.go#L55